packages/translations-provider/README.md

```diff
@@ -0,0 +1,114 @@
+# Translations Provider
+
+## Installation
+
+Using [yarn](https://yarnpkg.com/lang/en/):
+
+    $ yarn add @tshio/translations-provider
```

Comment:
Package name should be changed to `@tshio/json-translation-provider`

Reply:
We will change it (with rest wrong packages names) after merge.